### DISCUSSION TITLE
Remove direct email links to avoid spam

### DISCUSSION THREAD

**Discussion by Robb-Fr** [clicepfl/clic-website#239] [Ideas]
As we receive a lot of mails (blocked by epfl quarantine gateway but still) to the most visible addresses (the one on the index page sponsoring.clic@epfl.ch and clic@epfl.ch), it might be cool to find a non-elegant but effective way to noise those.

As we can see on many websites, we could change sponsoring.clic@epfl.ch -> sponsoring.clic AT epfl.ch
This seems to become more and more standard

**Comment by mochiruntime** [comment]
This would be a great idea, unfortunately it's an eternal tradeoff between usability and privacy. Any publicly visible mail address will get spammed, and obfuscating these addresses may lead to missed opportunities (non-tech people giving up, typos, etc).
IMO as long as the antispam protection does its job we should keep it that way, so that we remain easily reachable. Another solution would be to simply add a contact form that we could protect with some captcha (with the caveat that there aren't that many good open source libraries, accessibility is also a concern), and remove any existing email address.

See https://www.npmjs.com/package/svg-captcha for future reference

**Comment by Robb-Fr** [comment]
Yup makes sense, thanks for the answer, indeed contact form might be great but personally I don't find it very appealing, at least not as much as directly writting emails, we can keep it that way

**Comment by mochiruntime** [comment]
Moving this to discussions, as it may be relevant in the future
